projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/style/base.css

```diff
@@ -3,3 +3,5 @@
 
     https://jupyterlab.readthedocs.io/en/stable/developer/css.html
 */
+
+@import url('vdkDialogs.css');
```

Comment:
nit: css and overall resource files tend to be minimised/optimized for performance purposes (download, parse and apply); the codacy catches you have a white space (new line) in the resource file that is redundant